masterlist.yaml

```diff
@@ -1709,6 +1709,17 @@ plugins:
     group: *fixesGroup
     msg: [ *alreadyInUFO4P ]
 
+  - name: 'ModdableGoodfeels.esp'
+    url: [ 'https://www.nexusmods.com/fallout4/mods/11500' ]
+    dirty:
+      - <<: *quickClean
+        crc: 0x24BDA937
+        util: '[FO4Edit v4.0.4](https://www.nexusmods.com/fallout4/mods/2737)'
+        itm: 3
+    clean:
+      - crc: 0x0FFB910B
+        util: 'FO4Edit v4.0.4'
```

Comment:
We exclude cleaning data for dirty plugins after they're cleaned due to volatility in the checksum.

```suggestion
```